automation/run-tests.sh

```diff
@@ -227,8 +227,16 @@ function modprobe_ovs {
     lsmod | grep -q ^openvswitch || modprobe openvswitch || { echo 1>&2 "Please run 'modprobe openvswitch' as root"; exit 1; }
 }
 
+function vdsm_tests {
+  git clone https://gerrit.ovirt.org/vdsm
```

Comment:
This might make it faster:

```suggestion
  git clone --depth 1 https://gerrit.ovirt.org/vdsm
```

Reply:
Done, thanks